src/Time/DateTime.elm

```diff
@@ -205,8 +202,8 @@ mkDateTime date time =
 
 -}
 makeDateTime : Date -> Int -> DateTime
-makeDateTime date offset =
-    DateTime { date = date, offset = offset }
+makeDateTime withDate withTime =
+    DateTime { date = withDate, offset = withTime }
```

Comment:
I like `withOffset` better in the above two lines than `withTime`.  `withTime` is already used in `mkDateTime` and using it again here is at best jarring, if not confusing.